docs/consume-packages/install-use-packages-nuget-cli.md

```diff
@@ -1,118 +1,120 @@
+For example, to install the `Newtonsoft.json` package to the *packages* subdirectory, use the following command:
 
-    ```cli
-    nuget install <packageID> -OutputDirectory packages
-    ```
+```cli
+nuget install Newtonsoft.Json -OutputDirectory packages
+```
 
-    To install the `Newtonsoft.json` package to the *packages* folder, use the following command:
+Instead of specifying a package to install, you can specify an existing *packages.config* file in the current or another directory. The `install` command installs all the packages listed in the *packages.config* file.
 
-    ```cli
-    nuget install Newtonsoft.Json -OutputDirectory packages
-    ```
+```cli
+nuget install packages.config
+```
 
-Alternatively, you can use the following command to install a NuGet package using an existing `packages.config` file to the *packages* folder. This does not add the package to your project dependencies, but installs it locally.
+For example, the following command installs all the packages listed in the *packages.config* file at *C:\myPath* to the *packages* subdirectory in the current directory:
 
 ```cli
-nuget install packages.config -OutputDirectory packages
+nuget install C:\myPath\packages.config -OutputDirectory packages
 ```
 
 ## Install a specific version of a package
 
-If the version is not specified when you use the [install](../reference/cli-reference/cli-ref-install.md) command, NuGet installs the latest version of the package. You can also install a specific version of a Nuget package:
+The `install` command installs the latest version of a package, unless you specify a different version. To install a specific version of a package, use the `-Version` option:
 
 ```cli
 nuget install <packageID | configFilePath> -Version <version>
 ```
 
-For example, to add version 12.0.1 of the `Newtonsoft.json` package, use this command:
+For example, to install version 12.0.1 of the `Newtonsoft.json` package, use:
 
 ```cli
 nuget install Newtonsoft.Json -Version 12.0.1
 ```
 
-For more information on the limitations and behavior of `install`, see [Install a package](#install-a-package).
-
-## Remove a package
-
-To delete one or more packages, delete the packages you want to remove from the *packages* folder.
-
-If you want to reinstall packages, use the `restore` or `install` command.
-
 ## List packages
 
-You can display a list of packages from a given source using the [list](../reference/cli-reference/cli-ref-list.md) command. Use the `-Source` option to restrict the search.
+Use the [list](../reference/cli-reference/cli-ref-list.md) command to display a list of packages. Use the `-Source` option to restrict the list.
 
 ```cli
 nuget list -Source <source>
 ```
 
-For example, list packages in the *packages* folder.
+For example, to list packages in the *packages* subdirectory, use:
 
 ```cli
-nuget list -Source C:\Users\username\source\repos\MyProject\packages
+nuget list -Source packages
```

Comment:
Opposite question to the previous. Testing showed no path is required here?

Reply:
@Mikejo5000 - Right. I think this was an example path. (VS projects don't have to be at source/repos.) Using just "packages" lists the packages in a subfolder named packages in the current folder. The folder name "packages" seems arbitrary, at least, any folder name works with the command. I don't know if folder name "packages" is necessary for some other functionality. If you use just `nuget list` you get all the packages at nuget.org (or default location).

Reply:
@Mikejo5000 @nkolev92 - I was incorrect, full path (or URL) is needed for `-Source`.